src/XRP/serializer.ts

```diff
@@ -77,6 +77,7 @@ interface PathElementJSON {
   currencyCode?: string
 }
 
+type CurrencyJSON = string
```

Comment:
does a naked string count as JSON?  news to me, but seems reasonable

Reply:
Yeah, I think this type is unnecessary since the code referencing this is returning hex in a string format.